docs/resources/aws_elasticloadbalancingv2_listeners.md

```diff
@@ -0,0 +1,60 @@
+---
+title: About the aws_elasticloadbalancingv2_listeners Resource
+platform: aws
+---
+
+# aws\_elasticloadbalancingv2\_listeners
+
+Use the `aws_elasticloadbalancingv2_listeners` InSpec audit resource to test the properties of a collection of AWS ElasticLoadBalancingV2 Listeners.
+
+## Syntax
+
+An `aws_elasticloadbalancingv2_listeners` resource block returns all ElasticLoadBalancingV2 Listeners.
+    describe aws_elasticloadbalancingv2_listener(listener_arn: "test_arn") do
+      its { should exist }
+    end
+    
+#### Parameters
+
+##### listener_arn _(required)_
+
+See also the [AWS documentation on ElasticLoadBalancingV2 Listener](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-listener.html).
+
+## Properties
+
+|Property | Description|
+| --- | --- |
+|listener_arns | The Amazon Resource Name (ARN) of the listener. |
```

Comment:
Does this accept a list of ARNs?

Reply:
Yes earlier it used it, Updated it to accept load_balancers_arn  so that it can list all listeners in it .
Thanks